L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryHexImp1.cc

```diff
@@ -54,10 +54,10 @@ void HGCalTriggerGeometryHexImp1::fillMaps(const es_info& esInfo)
 {
     //
     // read module mapping file
-    std::map<short, short> wafer_to_module_ee;
-    std::map<short, short> wafer_to_module_fh;
-    std::map<short, std::map<short,short>> module_to_wafers_ee;
-    std::map<short, std::map<short,short>> module_to_wafers_fh;
+    std::unordered_map<short, short> wafer_to_module_ee;
+    std::unordered_map<short, short> wafer_to_module_fh;
+    std::unordered_map<short, std::unordered_map<short,short>> module_to_wafers_ee;
+    std::unordered_map<short, std::unordered_map<short,short>> module_to_wafers_fh;
```

Comment:
Actually it just reminds me that I need an ordered iterator for the second map, e.g. line [107](https://github.com/jbsauvan/cmssw/blob/1955deb2d2585910130254a7084fba3eb33d7936/L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryHexImp1.cc#L107) (but not for the first one), so I would need unordered_map\<short, map\<short,short\>\> (or multimap\<short, pair\<short,short\>\> should work as well, but would probably be slower when searching for a module line [132](https://github.com/jbsauvan/cmssw/blob/1955deb2d2585910130254a7084fba3eb33d7936/L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryHexImp1.cc#L132)).